user_manual/docs/rdm-responder.md

```diff
@@ -1,6 +1,159 @@
-RDM Responder Mode
+RDM Responder  {#responder}
 ================
 
 [TOC]
 
 # Overview {#responder-overview}
+
+The OLE software can simulate different types of RDM Device Models. This
+allows RDM controllers to be tested without having to purchase a many
+different types of RDM products.
+
+Each Device Model is identified with a 16-bit Device Model ID. See Section
+10.5.1 of the RDM Standard for more information on Device Model IDs.
+
+The device can only operate as one model at a time.
+
+Model ID (hex)  | Description
+--------------- | -------------
+0x0100          | LED Driver
+0x0101          | Proxy, which ACK_TIMERs all requests to child devices.
+0x0102          | Moving Light
+0x0103          | Sensor only device (no DMX footprint)
+0x0104          | E1.37-2 Network Device
+0x0105          | Dimmer, with sub-devices & status messages.
+
+## Selecting the Device Model ID {#responder-changing-models}
+
+The RDM Device Model is controlled by Manufacturer Specific RDM PIDs. They can
+be changed with any RDM Controller software that supports setting Manufacturer
+Specific PIDs. These PIDs do not appear in SUPPORTED_PARAMETERS, so in a sense
+they are 'hidden'.
+
+### MODEL_ID_LIST
+
+[MODEL_ID_LIST](http://rdm.openlighting.org/pid/display?manufacturer=31344&pid=32771)
+will return a list of Device Model IDs. The request PDL is 0.
```

Comment:
PDL is a bit advanced for users, I'd remove it, or word it that you can only get or something.